app/assets/stylesheets/components/common/_collapsable-mobile.scss

```diff
@@ -0,0 +1,26 @@
+.collapsable-mobile {
+
+  .collapsable__trigger-icon {
+    float: right;
+    margin: 10px 20px 0 5px;
+
+    @include respond-to($mq-m) {
+      display: none;
+    }
+  }
+
+  button {
```

Comment:
Can we use BEM here instead of the element? Also, can we remove the nesting and make the class names consistent with the component name?